src/dbt_score/config.py

```diff
@@ -1,13 +1,18 @@
 """This module is responsible for loading configuration."""
 
 import logging
-import tomllib
+import sys
 from dataclasses import dataclass, field, replace
 from pathlib import Path
 from typing import Any, Final
 
 from dbt_score.rule import RuleConfig
 
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
Is it a transitive dependency? We probably should have this in `pyproject.toml` for Python < 3.11?

Reply:
I think we need something like:

`tomli = { version = "...", python = "<3.11" }` in pyproject indeed

Reply:
Yes, good point 👍